[Bug] [scaleph-api] error in docker-compose

### Already searched before asking?

- [X] I had searched in the [issues](https://github.com/flowerfine/scaleph/issues?q=is%3Aissue+label%3A%22bug%22) and found no similar issues.


### Scaleph Version or Branch
dev branch

### What happened
when use docker-compose to deploy,the following error:
2023-08-09 00:21:09 Error: Could not find or load main class .opt.scaleph.libs.HikariCP-4.0.3.jar
2023-08-09 00:21:09 Caused by: java.lang.ClassNotFoundException: /opt/scaleph/libs/HikariCP-4/0/3/jar

### Error Exception

_No response_

### Screenshots

_No response_

### Are you willing to submit PR?

- [ ] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.contributor-covenant.org/version/2/1/code_of_conduct/)

--- a/scaleph-dist/src/assembly/scaleph-dist.xml
+++ b/scaleph-dist/src/assembly/scaleph-dist.xml
@@ -23,7 +23,7 @@
         <format>tar.gz</format>
     </formats>
     <includeBaseDirectory>true</includeBaseDirectory>
-    <baseDirectory>${project.parent.build.finalName}-${project.parent.version}</baseDirectory>
+    <baseDirectory>scaleph</baseDirectory>
 
     <fileSets>
         <fileSet>

--- a/scaleph-dist/src/bin/scaleph.sh
+++ b/scaleph-dist/src/bin/scaleph.sh
@@ -54,13 +54,17 @@ function start() {
     local JAVA_CMD="${JAVA_HOME}/bin/java"
   fi
   if [[ $1 == 0 ]]; then
+    shift
     echo "Staring in backend"
-    nohup $JAVA_CMD $JAVA_OPT >/dev/null 2>&1 &
+    nohup $JAVA_CMD $JAVA_OPT $ENV_PARAMS $@ >/dev/null 2>&1 &
     local pid=$!
     echo "Running with pid ${pid}"
     echo "${pid}" >${PID_FILE}
   else
-    $JAVA_CMD $JAVA_OPT
+    shift
+    echo "JAVA_OPTS=${JAVA_OPT}"
+    echo "ARGUMENTS=$ENV_PARAMS $@"
+    $JAVA_CMD $JAVA_OPT $ENV_PARAMS $@
   fi
 }
 
@@ -95,15 +99,23 @@ function print_help() {
 
 case $1 in
 start)
-  start 0
+  shift
+  start 0 $@
   ;;
 start-frontend)
-  start 1
+  shift
+  start 1 $@
+  ;;
+start-foreground)
+  shift
+  start 1 $@
   ;;
 stop)
+  shift
   stop
   ;;
 status)
+  shift
   status
   ;;
 *)

--- a/tools/docker/build/scaleph-api/Dockerfile
+++ b/tools/docker/build/scaleph-api/Dockerfile
@@ -1,16 +1,7 @@
 ARG BASE_RELEASE_IMAGE=eclipse-temurin:11-focal
-FROM maven:3.8-eclipse-temurin-11 as build
-
-COPY ./ /scaleph
-RUN cd /scaleph && \
-    tar -zxf /scaleph/scaleph-dist/target/scaleph-*-bin.tar.gz -C /scaleph/scaleph-dist/target  && \
-    mkdir /dist && \
-    cp -r /scaleph/scaleph-dist/target/scaleph-*/* /dist
-
-
-# BASE_RELEASE_IMAGE argument must be defined twice
-ARG BASE_RELEASE_IMAGE=eclipse-temurin:11-focal
 FROM $BASE_RELEASE_IMAGE as release
+
+ADD scaleph-dist/target/scaleph-*-bin.tar.gz /opt
 
 ENV MYSQL_URL="jdbc:mysql://mysql:3306/scaleph?useUnicode=true&characterEncoding=UTF-8&serverTimezone=Asia/Shanghai"
 ENV MYSQL_USERNAME=root
@@ -49,8 +40,6 @@ RUN mkdir -p $SCALEPH_HOME
 
 WORKDIR $SCALEPH_HOME
 
-COPY --from=build /dist $SCALEPH_HOME/
-
 EXPOSE 8080 8080
 
-ENTRYPOINT ["sh","-c","java -cp /opt/scaleph/libs/* cn.sliew.scaleph.Application $ENV_PARAMS"]
+ENTRYPOINT [ "sh", "-c", "/opt/scaleph/bin/scaleph.sh start-foreground" ]

